visearch-android/src/main/java/com/visenze/visearch/android/api/impl/SearchOperationsImpRetrofit.java

```diff
@@ -0,0 +1,51 @@
+package com.visenze.visearch.android.api.impl;
+
+import android.content.Context;
+
+import com.visenze.visearch.android.ColorSearchParams;
+import com.visenze.visearch.android.IdSearchParams;
+import com.visenze.visearch.android.UploadSearchParams;
+import com.visenze.visearch.android.ViSearch;
+import com.visenze.visearch.android.api.SearchOperations;
+import com.visenze.visearch.android.network.SearchService;
+
+public class SearchOperationsImpRetrofit implements SearchOperations {
```

Comment:
is there any difference between SearchOperationsImpRetrofit and previous SearchOperationsImp other than cancelSearch method ? Did not see anything require Retrofit here. If this is the case, pls rename to the previous name.

Reply:
oh nvm, i just saw previous implementation. Still pls rename to remove `Retrofit` from name. Look quite weird.

Reply:
changed already.